connection.go

```diff
@@ -62,15 +64,55 @@ func NewConnection(addr, user, pass string) *Connection {
 	}
 }
 
+func NewTLSConnection(addr, user, pass string, certPool *x509.CertPool) *Connection {
+	connection := NewConnection(addr, user, pass)
+	connection.dial = func(network, address string) (net.Conn, error) {
+		conn, err := net.DialTimeout(network, address, 5*time.Second)
+		if err != nil {
+			return nil, err
+		}
+
+		// TODO Save Info packet somewhere accessible
```

Comment:
We'll remove the TODO for now because the non-tls connection is also swallowing the INFO msg atm.